content/en/docs/concepts/metrics/_index.md

```diff
@@ -1,28 +1,190 @@
 ---
-title: Falco Metrics 
-description: Leverage continuous metrics for valuable insights into Falco's performance
-linktitle: Falco Metrics
-weight: 10
+title: Metrics
+description: Leverage continuous metrics for valuable insights into Falco's performance.
+weight: 50
+aliases:
+- ../metrics
+- ../metrics/falco-metrics
 ---
 
-Falco's optional native metrics framework helps you verify that Falco is running smoothly in production. It lets you check if Falco is working as expected, track the progress and stability of rollouts, and review internal metrics for potential configuration improvements. This way, you can confirm Falco's performance and make adjustments as needed for optimal operation.
+## Overview
 
-To explore the metrics functionality in more detail, refer to the [falco.yaml](https://github.com/falcosecurity/falco/blob/master/falco.yaml) config file and read the advanced Falco logging, alerting, and metrics sections (e.g. `metrics`).
+Falco’s native metrics framework provides real-time insights into your deployment’s performance. With these metrics, you can:
 
-Read [Prometheus Support](/docs/metrics/falco-metrics/#prometheus-support) to learn how to consume metrics via Prometheus.
+- **Monitor Health:** Verify that Falco is running smoothly in production.
+- **Track Performance:** Observe event rates, resource usage, and more.
+- **Troubleshoot Issues:** Identify potential configuration improvements based on detailed internal metrics.
+- **Detections Metrics:** Get insights into rules matching abnormal behavior, potential security threats, and compliance violations.
 
-Falco metrics are not enabled by default. The following are all the metrics config options available in the [falco.yaml](https://github.com/falcosecurity/falco/blob/master/falco.yaml) config file:
+{{% pageinfo color="info" %}}
+As a real-world example of the potential these metrics can offer, we invite you to examine the Grafana dashboards of live Falco deployments for our testing infrastructure at [monitoring.prow.falco.org](https://monitoring.prow.falco.org/).
+{{% /pageinfo %}}
+
+{{% pageinfo color="info" %}}
+For performance-related guidance, take a look at our [troubleshooting section](/docs/troubleshooting/performance).
+{{% /pageinfo %}}
+
+## Configuration {#configuration}
+
+Falco metrics are disabled by default. To start using them, you must enable the `metrics` option in your configuration (defined in the [falco.yaml](https://github.com/falcosecurity/falco/blob/master/falco.yaml). Further options allow you to customize the metrics functionality to meet your monitoring needs.
 
 ```yaml
+# [Stable] `metrics`
```

Comment:
This looks hard to maintain, we removed explicit references to yaml for this reason

Reply:
I know, but I can't find a better solution, and the comments in the yaml are pretty descriptive.

Do you have any other solution? :thinking:

Reply:
I think we could avoid copy-pasting the entire file and just keep some relevant snippets so people know where to change

Reply:
essentially, remove the commented block below and keep from `metrics:`